PubNub/Network/Parsers/PNSubscribeParser.m

```diff
@@ -274,25 +270,28 @@ + (NSMutableDictionary *)messageFromData:(id)data
 
 + (NSMutableDictionary *)presenceFromData:(NSDictionary *)data {
     
+#warning ask about how this is affected
+    NSDictionary *payload = data[@"d"];
```

Comment:
@jzucker2 Because data[kPNMessagePayloadDataKey] will be passed from calling method, there is no need in this and this line can be reverted to original one.